### DISCUSSION TITLE
ETH transfer but not received

### DISCUSSION THREAD

**Discussion by Imtiazhussain911** [blockscout/blockscout#11465] [General]
Hello sir two days ago I transferred 0.0045ETH to My OKX wallet my fund not received in OKX please resolve my issue as soon as possible 
![Uploading Screenshot_20241217_164333.jpg…]()

**Comment by Imtiazhussain911** [comment]
Please resolve my issue as soon as possible 😭

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
